template/Dockerfile

```diff
@@ -17,11 +17,10 @@ RUN usermod "--login=${NB_USER}" "--home=/home/${NB_USER}" --move-home "-u ${NB_
     echo "${NB_USER}" > "/etc/arg_mamba_user" && \
     :
 ENV MAMBA_USER=$NB_USER
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
I think the problem with this is that this variable will also be present inside the image after it is built. I am wondering if `ARG` would be a better choice such that we use this only at build time and not persist later.